awx/main/conf.py

```diff
@@ -845,6 +850,7 @@
     help_text=_('Enable preview of new user interface.'),
     category=_('System'),
     category_slug='system',
+    hidden=True,
```

Comment:
The one thing I'm not getting from this is how this list of settings to hide came about. Will this just lead to complaints that we hid these?